test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
- [ ] As stated the project requirement Kindly make sure that `test.md` file is actually ignored by git and is not present in your GitHub repository.

![Capture](https://user-images.githubusercontent.com/93598189/211783152-2ab17b7d-bec3-4fad-a8f0-c917f7ef9fbb.PNG)

Reply:
Fixed.